### PR TITLE
slsa-verifier: init at 2.0.1

### PR DESCRIPTION
**Pull request by developer-guy**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

PTAL @asraa @laurentsimon @ianlewis

### CODE REVIEW

File: pkgs/tools/security/slsa-verifier/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  pname = "slsa-verifier";
+  version = "2.0.1";
+
+  src = fetchFromGitHub {
+    owner = "slsa-framework";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-Gef8TQSd6bTWIzFOQ9xjqB49We7IKBu9p/Lb426nNbc=";
+  };
+  vendorSha256 = "sha256-1syIEjvqYHCiOLf8Fc2vghFKfN6ADM05By11jGNZODs=";
```
Comment (AndersonTorres) [inline pkgs/tools/security/slsa-verifier/default.nix:13]:
```suggestion

  vendorSha256 = "sha256-1syIEjvqYHCiOLf8Fc2vghFKfN6ADM05By11jGNZODs=";
```

File: pkgs/tools/security/slsa-verifier/default.nix [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "slsa-verifier";
+  version = "2.0.1";
+
+  src = fetchFromGitHub {
+    owner = "slsa-framework";
+    repo = pname;
```
Comment (AndersonTorres) [inline pkgs/tools/security/slsa-verifier/default.nix:9]:
```suggestion
    repo = "slsa-verifier";
```

File: pkgs/tools/security/slsa-verifier/default.nix [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{ lib, buildGoModule, fetchFromGitHub }:
```
Comment (AndersonTorres) [inline pkgs/tools/security/slsa-verifier/default.nix:1]:
```suggestion
{ lib
, fetchFromGitHub 
, buildGoModule
}:
```

File: pkgs/tools/security/slsa-verifier/default.nix [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+  CGO_ENABLED = 0;
+  GO111MODULE = "on";
+  GOFLAGS = "-trimpath";
+
+  subPackages = [ "./cli/slsa-verifier" ];
+
+  tags = [ "netgo" ];
+
+  ldflags = [ "-s" "-w" "-buildid=" "-X sigs.k8s.io/release-utils/version.gitVersion=${version}" ];
```
Comment (AndersonTorres) [inline pkgs/tools/security/slsa-verifier/default.nix:23]:
```suggestion
  ldflags = [ 
    "-s"
    "-w" 
    "-buildid=" 
    "-X sigs.k8s.io/release-utils/version.gitVersion=${version}" 
  ];
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
Reword the commit message as

```
slsa-verifier: init at 2.0.1
```

**Review by AndersonTorres — APPROVED** [review]

**Comment by wegank** [comment]
Result of `nixpkgs-review pr 220670` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>slsa-verifier</li>
  </ul>
</details>

**Comment by wegank** [comment]
@ofborg build slsa-verifier slsa-verifier.passthru.tests

File: pkgs/tools/security/slsa-verifier/default.nix [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  subPackages = [ "cli/slsa-verifier" ];
+
+  tags = [ "netgo" ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-buildid="
+    "-X sigs.k8s.io/release-utils/version.gitVersion=${version}"
+];
```
Comment (wegank) [inline pkgs/tools/security/slsa-verifier/default.nix:32]:
```suggestion
  ];
```

**Review by wegank — APPROVED** [review]

File: pkgs/tools/security/slsa-verifier/default.nix [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+  version = "2.0.1";
+
+  src = fetchFromGitHub {
+    owner = "slsa-framework";
+    repo = "slsa-verifier";
+    rev = "v${version}";
+    sha256 = "sha256-Gef8TQSd6bTWIzFOQ9xjqB49We7IKBu9p/Lb426nNbc=";
+  };
+
+  vendorSha256 = "sha256-1syIEjvqYHCiOLf8Fc2vghFKfN6ADM05By11jGNZODs=";
```
Comment (AndersonTorres) [inline pkgs/tools/security/slsa-verifier/default.nix:17]:
```suggestion
    hash = "sha256-Gef8TQSd6bTWIzFOQ9xjqB49We7IKBu9p/Lb426nNbc=";
  };

  vendorHash = "sha256-1syIEjvqYHCiOLf8Fc2vghFKfN6ADM05By11jGNZODs=";
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
Besides, LGTM

File: pkgs/tools/security/slsa-verifier/default.nix (hunk 6 above)
Comment (developer-guy) [reply pkgs/tools/security/slsa-verifier/default.nix:17]: thanks, fixed.

File: pkgs/tools/security/slsa-verifier/default.nix (hunk 5 above)
Comment (developer-guy) [reply pkgs/tools/security/slsa-verifier/default.nix:32]: thanks, fixed.

Comment (wegank) [reply pkgs/tools/security/slsa-verifier/default.nix:32]: Not really...?

**Pull request merged**